README.md

```diff
@@ -42,42 +42,18 @@ Pre-built binaries are available for download on [GitHub Releases](https://githu
 The following build procedure works on Linux:
 
 ```
-sudo su
-sudo apt-get install python-serial # Otherwise cannot build for ESP32
-
-BD=esp8266:esp8266:d1_mini:xtal=80,eesz=4M3M
-# or 
-BD=esp32:esp32:esp32
-
-wget http://downloads.arduino.cc/arduino-1.8.8-linux64.tar.xz
-tar xf arduino-*-linux64.tar.xz
-export PATH=$(readlink -f arduino-*/):$PATH
-
-arduino --pref "boardsmanager.additional.urls=http://arduino.esp8266.com/stable/package_esp8266com_index.json,https://dl.espressif.com/dl/package_esp32_index.json" --save-prefs
-if [[ "$BD" =~ "esp8266:esp8266:" ]]; then arduino --install-boards esp8266:esp8266:2.5.0 ; fi
-if [[ "$BD" =~ "esp32:esp32:" ]]; then arduino --install-boards esp32:esp32 ; fi
-arduino --pref "compiler.warning_level=all" --save-prefs
-mkdir -p $HOME/Arduino/libraries/
-cd $HOME/Arduino/libraries/
-git clone https://github.com/greiman/SdFat
-git clone -o 95dedf7 https://github.com/me-no-dev/ESPAsyncWebServer
-git clone -o 7e9ed22 https://github.com/me-no-dev/ESPAsyncTCP # for esp8266
-git clone https://github.com/me-no-dev/AsyncTCP # for esp32
-git clone -o 1c02154 https://github.com/alanswx/ESPAsyncWiFiManager
-( git clone -b 6.x https://github.com/bblanchon/ArduinoJson ; cd ArduinoJson ; git reset --hard 3df4efd )
-cd -
-
-git clone https://github.com/probonopd/WirelessPrinting/
-cd WirelessPrinting/
-
-VERSION=$(git rev-parse --short HEAD)
-HERE=$(readlink -f .)
-sed -i -e 's|#define SKETCH_VERSION ".*"|#define SKETCH_VERSION "'$VERSION'"|' $PWD/ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino
-arduino --pref build.path=. --verify --verbose-build --board $BD ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino
-BOARD=$(echo $BD | cut -d ":" -f 3)
-mv ./ESP8266WirelessPrintAsync.ino.bin "ESP8266WirelessPrintAsync_${BOARD}_${VERSION}.bin"
-
+# Get PlatformIO (the toolchain we use for compiling)
+git clone https://github.com/probonopd/WirelessPrinting
+cd WirelessPrinting
+wget -c https://downloads.egenix.com/python/install-pyrun
```

Comment:
There's no reason to use a (potentially) non-free Python distribution just to get PlatformIO running. Python 3 is usually available anyway. Even if `pip` was missing, you could just run `python3 -m ensurepip`, then run `pip` like `python3 -m pip install ...`.

Also, why even bother including this? A link to https://platformio.org/install/cli or https://docs.platformio.org/en/latest/installation.html is sufficient.

You're making this overly complicated.

Reply:
I always want to boot any random (usually LTS) Linux distribution, and copy&paste something, and have it work. Without distribution-specific instructions that always seem to be for distributions I am not running at that moment. And without having to read up about how to install some auxiliary tools. From my experience, these instructions should be pretty much copy-and-paste-able on most systems. I love them.

Reply:
It's highly confusing to random users, though. And bad style. Especially the use of a weird name like `appdir`. I think it's misleading. But it's your docs, I don't really care too much.

If you think PlatformIO's instructions are not sufficient, why not rather try to help there?

Reply:
Agree that it's not really a full-fledged AppDir, hence will change the name...